BritishGeologicalSurvey/AGS-Validator
# Ags convert

This merge request includes:

+ Add the `convert` method
+ Revise functions to return log output as a string
+ Add file size and timestamp to our generated metadata
+ Add `is_valid` functon

Note that there is also an `is_log_valid` function.  This could be useful if you want to include the validity status in the JSON response from the `validate` endpoint.

### To test

It's a bit fiddly to run the function individually by hand, and I'm not sure if the API is wired up, but you can run the tests `pytest -vs test` and check the output to confirm that they work.  If the API is wired up, then confirm that it works for `real/A3040_03.ags`.
